perses/app/cli.py

```diff
@@ -83,7 +84,7 @@ def _test_platform(platform_name):
 
 
 @click.command()
-@click.option("--yaml", type=click.Path(exists=True, dir_okay=False), required=True,
+@click.option("--yaml", type=click.Path(path_type=AnyPath), required=True,
```

Comment:
I had to remove them since they don't support a path like `s3://perses-testing/s3_test.yaml`. I liked them as well, now the path gets casted to an `AnyPath` and if that fails, then it will throw an error at least.